CHANGELOG.unreleased.md

```diff
@@ -12,3 +12,5 @@
 
 - xo-server v5.40.0
 - xo-web v5.40.0
+- vhd-lib v0.6.2
+- xo-vmdk-to-vhd v0.1.8
```

Comment:
Packages are released in order, please keep dependencies above.

Reply:
wait, I'm pretty sure I got scolded for putting stuff above before.

Reply:
I cannot find such a comment but maybe it was the order in between your packages that was incorrect.